web/styles/input_pill.css

```diff
@@ -205,6 +205,13 @@
 .panel_user_list > .pill-container,
 .stream_creator_details > .display_only_pill {
     background-color: hsl(0deg 0% 0% / 7%);
+    gap: 2px;
+    flex-wrap: nowrap;
+
+    &.inline_with_text_pill > .pill-deactivated {
+        font-size: 0.9em;
+        padding-right: 2px;
```

Comment:
Have kept the `font-size` same as what we had before for the icon in deactivated users list.